ffi/mupdf.lua

```diff
@@ -586,12 +586,16 @@ function page_mt.__index:getPageLinks()
             x0 = link.rect.x0, y0 = link.rect.y0,
             x1 = link.rect.x1, y1 = link.rect.y1,
         }
-        local page = M.fz_resolve_link(context(), link.doc, link.uri, nil, nil)
+        local pos = ffi.new("float[?]", 2)
```

Comment:
```suggest
        local pos = ffi.new("float[2]")
```

This isn't actually a VLA, in that we already know the amount of elements we want ;).